src/Sarif.WorkItems/SarifWorkItemModel.cs

```diff
@@ -54,9 +56,7 @@ public SarifWorkItemModel(SarifLog sarifLog, SarifWorkItemContext context = null
                 Text = JsonConvert.SerializeObject(sarifLog, Formatting.Indented),
             };
 
-            string title = sarifLog.Runs?[0]?.CreateWorkItemTitle();
-
-            this.Title = title ?? "[ERROR GENERATING TITLE]";
+            this.Title = sarifLog.Runs?[0]?.CreateWorkItemTitle();
```

Comment:
>Title [](start = 17, length = 5)

Shouldn't we validate the fields at some point? Perhaps not here, but maybe in the FilingClient?

Reply:
Yes. And emit results to telemetry.